### PR TITLE
Tidy up word wrapping to help with PDF layout.

### PR DESCRIPTION
**Pull request by rhattersley**
Gets rid of some unwanted whitespace in the PDF - e.g. the first couple of paragraphs of section 3.5, "Flags".

### CODE REVIEW

File: appf.adoc [hunk 1]
```diff
@@ -4,167 +4,122 @@
...
+Each section contains: the valid name that is used with the
+**`grid_mapping_name`** attribute; a list of the specific attributes
+that may be used to assign values to the mapping's parameters; the
+standard names used to identify the coordinate variables that contain
+the mapping's independent variables; and references to the mapping's
+definition or other information that may help in using the mapping.
+Since the attributes used to set a mapping's parameters may be shared
+among several mappings, their definitions are contained in a table in
+the final section. The attributes which describe the ellipsoid and prime
+meridian may be included, when applicable, with any grid mapping.
```
Comment (marqh) [inline appf.adoc]: Explicitly breaking these lines appears to make no difference to the pd for html output.  Only lines 8 and 9 with their leading space have any formatting effect.
We've not enforced a maximum line length in the content overall.

So, do you think that this line breaking adds readability for the editors?
Is this the intent of the change, or is the change only targeting the type spacing of the PDF output?

(similarly for numerous changes in this PR)

Comment (rhattersley) [inline appf.adoc]: Thanks for taking a look @marqh.

The change is driven by the need to fix the layout in the PDF output, but improving readability & maintainability was an intended side-effect. For example, the shorter lines should help when showing diffs on GitHub.

Comment (marqh) [inline appf.adoc]: > The change is driven by the need to fix the layout in the PDF output, but improving readability & maintainability was an intended side-effect. For example, the shorter lines should help when showing diffs on GitHub.

that's fine by me.  i've made conflicts again, so please rebase for me

thank you

**Comment by rhattersley** [comment]
Rebased.

**Comment by marqh** [comment]
excellent, thank you

**Pull request merged**